sql/rename.go

```diff
@@ -145,6 +145,7 @@ func (p *planner) RenameTable(n *parser.RenameTable) (planNode, error) {
 	}
 
 	tableDesc.SetName(n.NewName.Table())
+	tableDesc.ParentID = targetDbDesc.ID
```

Comment:
added test for parentID and name being set/changed properly on `CREATE` and `RENAME`.